src/Input/Select/Select.tsx

```diff
@@ -122,6 +122,18 @@ const Select: ISelect = (props: Props) => {
     return () => document.removeEventListener('click', onClickOutside, false);
   }, []);
 
+  React.useEffect(
+    function indicateBodyIsClickable() {
+      if (isFocus) {
+        // This indicates the document is clickable, so that onClickOutside works on ios
+        document.body.style.cursor = 'pointer';
+      }
+
+      return () => document.body.removeAttribute('style');
```

Comment:
on desktop this will make the whole page has pointer cursor, and we can not make sure we can do this `document.body.removeAttribute('style')`

please try other solution.